### PR TITLE
Adding tests

### PR DESCRIPTION
**Pull request by mgmart**
Some additional tests (and a small change to Readme)

Did the pull request at this stage because I don't think I will be able to contribute this year.

### CODE REVIEW

File: MobileOrg.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1109,7 +1116,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
-				DEVELOPMENT_TEAM = 59B7VFRF7R;
+				DEVELOPMENT_TEAM = 9VWR9JYXC6;
```
Comment (webframp) [inline MobileOrg.xcodeproj/project.pbxproj]: changing DEVELOPMENT_TEAM is going to be a problem

File: README.org [hunk 2]
```diff
@@ -44,9 +44,11 @@
 ** Setting up your environment
 
   For the usage of the Dropbox API an application key is
-  necessary. You can obtain it [[https://www.dropbox.com/developers/apps/create][from Dropbox.]]  Find the file
-  AppKey.plist in the project and obtain the information you've
-  received from Dropbox there.
+  necessary. You can obtain it [[https://www.dropbox.com/developers/apps/create][from Dropbox.]] It is important to apply
```
Comment (webframp) [inline README.org]: a good note for new devs

**Comment by webframp** [comment]
@mgmart the extra tests are really great. Only concern is the dev team identifier change.

**Comment by mgmart** [comment]
@webframp That's a Xcode 8 problem. We do have to switch the identifier back and forth also for local testing/running.
 
There is probably a solution described [here](http://stackoverflow.com/a/4042489). Which could allow us to ignore the hunks more easily. But this would lead to a complicated workflow.

The solution described [here](https://livecode.com/how-to-create-a-free-ios-development-provisioning-profile/) would help with running/testing but would not solve the problem for distribution.

There is also a good write-up [here](https://pewpewthespells.com/blog/migrating_code_signing.html#building-for-development-xcode-8)

I could imagine a git-hook which sets the identifier accordingly to the local settings.

**My suggestion would be that we live with switching back and forth the identifier for the time being and we provide a solution for this problem as soon as possible.**

**Comment by webframp** [comment]
I'm fine with that, I'll merge for now and change it back later. We could also just use `git add -p` to do a patch add and skip those chunks in the pbxproj that modify the value of DEVELOPMENT_TEAM

**Pull request merged**